src/kcc/api/serializers.py

```diff
@@ -2,12 +2,38 @@
 
 from rest_framework import serializers
 
+from kcc.datamodel.models import ContactMoment, Klant
 
 logger = logging.getLogger(__name__)
 
 
-class ExampleSerializer(serializers.HyperlinkedModelSerializer):
+class KlantSerializer(serializers.HyperlinkedModelSerializer):
     class Meta:
-        model = ...
-        fields = ("url", ...)
-        extra_kwarsg = {"url": {"lookup_field": "uuid"}}
+        model = Klant
+        fields = (
+            "url",
+            "voornaam",
+            "achternaam",
+            "adres",
+            "telefonnummer",
+            "emailadres",
+        )
+        extra_kwargs = {"url": {"lookup_field": "uuid"}}
+
+
+class ContactMomentSerializer(serializers.HyperlinkedModelSerializer):
+    class Meta:
+        model = ContactMoment
+        fields = (
+            "url",
+            "klant",
```

Comment:
Klant should have the usual help text that goes for relations:

URL-referentie naar een KLANT ...